fpga/.gitignore

```diff
@@ -0,0 +1,3 @@
+*
```

Comment:
I think we should make the builds in here look more standard so we can change this ignore all to something like `generated-src`,`output`, etc.

Reply:
Reasonable goal.